packages/embark/src/lib/modules/deployment/index.js

```diff
@@ -100,10 +100,10 @@ class DeployManager {
                 return done(_err);
               }
               if (contracts.length === 0) {
-                self.logger.info(__("no contracts found"));
+                self.logger.info(__("None of your Smart Contracts needed compilation, nothing to deploy."));
```

Comment:
Updated this message as "no contracts found" actually doesn't apply in this case.